### ISSUE TITLE
Profiler API and Service are not available

### ISSUE THREAD

**Issue by AlexisBRENON** [tensorflow/tensorflow#34823]
## URL(s) with the issue:

https://www.tensorflow.org/tensorboard/tensorboard_profiling_keras#other_ways_for_profiling

## Description of the issue (what needs changing):

The guide presents the profiler API and service as other ways of profiling.
Both of these use modules inside the `tensorflow.python` package.
However, this package does not seem to exist as exhibited by the error message:

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-14-c7b3d51d1060> in <module>()
----> 1 with tf.python.eager.profiler.Profiler('logdir_path'):
      2   # do your training here
      3   pass
      4 
      5 

AttributeError: module 'tensorflow' has no attribute 'python'
```

### Submit a pull request?

I would be pleased to submit a pull request, but do not know if there is any TF2.0 compatible way to use profile API or service.

**Comment by gowthamkpr** [comment]
Was able to reproduce the issue. Please find the gist [here](https://colab.sandbox.google.com/gist/gowthamkpr/d2ddb1cfc3be57ae387be5fcabab8e92/copy-of-external-tensorboard_profiling_keras.ipynb). Thanks!

**Comment by qiuminxu** [comment]
@AlexisBRENON  Can you try:
from tensorflow.python.eager import profiler
# Context manager APIs
with profiler.Profiler('logdir_path'):
  # do your training here
  pass

**Comment by AlexisBRENON** [comment]
@qiuminxu This seems to work. However, is it not fragile to import something from `tensorflow.python`?

**Comment by qiuminxu** [comment]
@AlexisBRENON, yes, ideally we want to have a tf.profiler public API, but that is not exposed yet.
The public API is under discussion, and hopefully we will have a version early next year.

**Comment by AlexisBRENON** [comment]
@qiuminxu Is there a place where I can follow the discussion and API progress?

**Comment by qiuminxu** [comment]
Hi @AlexisBRENON, sorry for the wait.
The public API is finally available in TF 2.2. https://www.tensorflow.org/versions/r2.2/api_docs/python/tf/profiler/experimental
They are tf.profiler.experimental.xxx for now, for the first release, and will be moved to tf.profiler in the next release. 

from tensorflow.python.eager import profiler continues to work in this release, but will be deprecated and move to the new API.

**Comment by AlexisBRENON** [comment]
Thanks for the feedback. I'll try that ASAP.
I suppose that I can close this issue.
